### PR TITLE
Add 'bluesky-queueserver' and 'bluesky-queueserver-api' to the landing page

### PR DESCRIPTION
**Pull request by dmgav**
This PR is more extensive version of https://github.com/bluesky/bluesky.github.io/pull/16

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,6 +234,32 @@ <h2>"Event Model" &mdash; Bluesky's Data Model</h2>
           <div class="col-md-5">
           </div>
         </div>
+        <hr class="featurette-divider" />
+        <div class="row">
+          <div class="col-md-7">
+            <h2>Bluesky Queue Server &mdash; Remote Control of Experiments</h2>
+            <p class="lead "> Bluesky Queue Server allows to create and remotely manage
```
Comment (danielballan) [inline index.html]:
```suggestion
            <p class="lead "> Bluesky Queue Server allows users to create and remotely manage
```

File: index.html [hunk 2]
```diff
@@ -41,6 +41,8 @@
                 <a class="dropdown-item" href="https://bluesky.github.io/bluesky-widgets">Bluesky Widgets (beta)&mdash; Graphical Components for Jupyter and Qt</a>
                 <a class="dropdown-item" href="https://bluesky.github.io/suitcase">Suitcase &mdash; Export / Serialization</a>
                 <a class="dropdown-item" href="https://bluesky.github.io/event-model">Event Model &mdash; Data Model & Schema</a>
+                <a class="dropdown-item" href="https://bluesky.github.io/bluesky-queueserver">Bluesky Queue Server &mdash; Server for Remote Control of Experiments</a>
```
Comment (danielballan) [inline index.html]: I think this should move higher in the list, above Tiled. (In fact, there is an entry already there.)

File: index.html [hunk 3]
```diff
@@ -232,6 +234,32 @@ <h2>"Event Model" &mdash; Bluesky's Data Model</h2>
           <div class="col-md-5">
           </div>
         </div>
+        <hr class="featurette-divider" />
+        <div class="row">
+          <div class="col-md-7">
+            <h2>Bluesky Queue Server &mdash; Remote Control of Experiments</h2>
```
Comment (danielballan) [inline index.html]: Same as with the menu above, I think Queueserver deserves a more prominent position here---above Widgets, Suitcase, and Event Model.

File: index.html (hunk 2 above)
Comment (dmgav) [reply index.html]: I reordered the menu items

File: index.html (hunk 3 above)
Comment (dmgav) [reply index.html]: I reordered the items. May be Databroker should be moved before the Suitcase or before the Queue Server?

Comment (dmgav) [reply index.html]: I also think that Bluesky Queue Server API should not be a separate large item, but it should be linked from the description in Bluesky Queue Server item.

**Pull request merged**